### PR TITLE
Add cases for libnbd is_zero

### PR DESCRIPTION
**Pull request by xiaodwan**
1) Add cases for libnbd is_zero, bz1809216
2) Reform libnbd.py and libnbd.cfg to support more test scenarios.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.libnbd.get_size: PASS (1.70 s)
(1/1) type_specific.io-github-autotest-libvirt.libnbd.is_zero: PASS (1.67 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh @yantian0 Please help to review this PR, thanks.

File: v2v/tests/src/libnbd/libnbd.py [hunk 1]
```diff
@@ -3,31 +3,81 @@
...
-            (expected_size, real_size))
-
-    handle.pwrite(expected_str, 0, nbd.CMD_FLAG_FUA)
-    real_str = handle.pread(len(expected_str), 0)
-
-    if real_str != expected_str:
-        test.fail(
-            'pwrite or pread failed: expected: %s, real: %s' %
-            (expected_str, real_str))
+    checkpoint = params.get('checkpoint')
```
Comment (chloerh) [inline v2v/tests/src/libnbd/libnbd.py:8]: Why not put this line under function definitions, closer to the other test steps?

Comment (xiaodwan) [reply v2v/tests/src/libnbd/libnbd.py:8]: hmm, I'm OK with it. 
I put it before all functions to keep same style with other files.

Comment (chloerh) [reply v2v/tests/src/libnbd/libnbd.py:8]: Of course, as long as it's consistent with other v2v code.

**Pull request merged**